crates/provider-sdk/src/lib.rs

```diff
@@ -120,6 +120,9 @@ pub struct LinkConfig<'a> {
 
     /// Configuration provided to the provider (either as the target or the source)
     pub config: &'a HashMap<String, String>,
+
+    /// WIT metadata for the link
+    pub wit_meta: (&'a WitNamespace, &'a WitPackage, &'a Vec<WitInterface>),
```

Comment:
Feels like a nit but I think it's valid, could we just call it `wit_metadata`? I think the longer form helps with parsing it in my brain. I would love to come up with a good name for what you call the ns, package, and interfaces but I can't think of one

Reply:
Will do -- yeah the longer form is also clearer